test/k8sT/Policies.go

```diff
@@ -2354,6 +2338,228 @@ var _ = SkipDescribeIf(func() bool {
 	})
 })
 
+// Create a new Describe block because we need to run some tests in GKE. For
+// example, the kube-apiserver policy matching feature needs coverage on GKE as
+// there are two cases for that feature:
+//   * kube-apiserver running within the cluster (Vagrant VMs)
+//   * kube-apiserver running outside of the cluster (GKE)
+var _ = SkipDescribeIf(helpers.DoesNotRunOn419OrLaterKernel,
+	"K8sPolicyTestExtended", func() {
+		var (
+			kubectl *helpers.Kubectl
+
+			// these are set in BeforeAll()
+			ciliumFilename string
+			daemonCfg      map[string]string
+		)
+
+		BeforeAll(func() {
+			kubectl = helpers.CreateKubectl(helpers.K8s1VMName(), logger)
+			daemonCfg = map[string]string{
+				"tls.secretsBackend": "k8s",
+				"debug.verbose":      "flow",
+				"hubble.enabled":     "true",
```

Comment:
I can speak for the last two, but not the first. The last two will help with the sysdump investigation if a test fails. These are the same options configured in the first Describe block as well.

Reply:
Not specific to this PR then, but maybe we should enable those options for all tests in `helpers/kubectl.go` rather than in specific `Describes`. `hubble.enabled=true` is already the default there. `debug.verbose` is sneakily set to `flow` in the definition of helper function `DeployCiliumAndDNS` :grimacing:

Reply:
Good point, I didn't notice this. I'll followup to clean this up when https://github.com/cilium/cilium/pull/18431 is merged. (If that merged before this one, then I'll rebase, otherwsie it'll just a followup PR.)